examples/src/examples/user-interface/text-auto-font-size.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import * as pc from 'playcanvas/build/playcanvas.js';
+import Example from '../../app/example';
+import { AssetLoader } from '../../app/helpers/loader';
+
+class TextAutoFontSizeExample extends Example {
+    static CATEGORY = 'User Interface';
+    static NAME = 'Text Auto Font Size';
+
+    load() {
+        return <>
+            <AssetLoader name='font' type='font' url='static/assets/fonts/courier.json' />
+        </>;
+    }
+
+    example(canvas: HTMLCanvasElement, assets: { font: pc.Asset }): void {
+
+        // Create the application with input and start the update loop
+        const app = new pc.Application(canvas, {
+            mouse: new pc.Mouse(document.body),
```

Comment:
is the input needed here?

Reply:
Same thing as the other example, I think it's nice to have.